hack/lib/build/binaries.sh

```diff
@@ -3,11 +3,25 @@
 # This library holds utility functions for building
 # and placing Golang binaries.
 
+# os::build::modvendorflag evaluates to either -mod=version or nothing
+function os::build::modvendorflag() {
+  tmpfile=${TMPDIR:-/tmp}/test${RANDOM}.go
+  trap 'rm -f "${tmpfile}"' EXIT
+  echo 'package main; func main() {}' > "${tmpfile}"
+  if go run -mod=vendor "${tmpfile}" ; then
+    echo "-mod=vendor"
+  fi
+}
+readonly -f os::build::modvendorflag
+
 # os::build::build_binary builds the openshift-builder binary
 function os::build::build_binary() {
   # Fetch the version.
   local version_ldflags
   version_ldflags=$(os::build::ldflags)
-  go build -ldflags "${version_ldflags}" -tags "${OS_GOFLAGS_TAGS-}" -o openshift-builder ./cmd
+  # Fetch additional build flags.
+  local mod_vendor_flag
+  mod_vendor_flag=$(os::build::modvendorflag)
+  go build ${mod_vendor_flag} -ldflags "${version_ldflags}" -tags "${OS_GOFLAGS_TAGS-}" -o openshift-builder ./cmd
```

Comment:
Note that on CI the golang 1.13 imagestream has the `-mod=vendor` flag set.

Reply:
Not sure I understand this.  Do you have a pointer to how it's done?  Is it done by setting GOFLAGS?  Should I drop this chunk?

Reply:
> Is it done by setting GOFLAGS?

Yes - I don't believe that flag is set in our golang 1.12 imagestream. In 1.13 `GOFLAGS="-mod=vendor` is set as an env var because of a golang bug. I don't see any harm setting it here in though.

Reply:
Makes sense.  The option isn't recognized by 1.12.